Some users install a web browser to look at HTML documentation when
they are offline.  In this case, it is not nice to have the default
page be a site that will time out.  Instead, we should use a local
"launch" page, similar to what Konqueror and Iceweasel do already.

Unfortunately there is no generic page of that sort, and I can't be
bothered writing one.  So for now, I will just make the homepage
Midori's user documentation.  This is similar to what Dillo did/does.

--- a/midori/midori-websettings.c
+++ b/midori/midori-websettings.c
@@ -558,7 +558,7 @@
                                      "homepage",
                                      _("Homepage"),
                                      _("The homepage"),
-                                     "http://www.google.com",
+                                     "file:///usr/share/doc/midori/user/midori.html",
                                      flags));
 
     /**
